TUTORIAL.md

```diff
@@ -213,17 +213,17 @@ let timerTick (dispatch: Dispatch<Msg>) =
+The final alias, `Cmd<'msg>`, is just a list of `Sub<'msg>`, i.e. a list of `Dispatch<'msg> -> unit` functions. In other words, the `update` function can return a list of `Dispatch<'msg> -> unit` functions that the MVU will execute by providing a dispatch function. These commands functions, as you saw above, can then dispatch any message at any time. Therefore, if you need to do impure stuff such as calling a web API, you simply create a function accepting `dispatch`, perform the work within it asynchronously, and then use the `dispatch` argument it will have been provided to insert the response into the MVU event loop.
 
-In other words, you don’t call the impure functions yourself; the MVU library calls them for you. Furthermore, from the point of view of your model, everything happens asynchronously (in the sense that your app and update loop continues without waiting on a response, and reacts to the “response” message when it arrives).
+In other words, unlike the model which is given to the view function for interpretation, the `Cmd<'msg>` function will be invoked by the MVU. From the point of view of your model, everything happens asynchronously: your app and UI update loop continues without waiting for the results of the `Cmds<'msg>` and when the commands complete the results are `dispatch` into the event loop as just another event.
 
 For example:
 
 * The user clicks a button to log in, which dispatches a `SignInRequested` message
-* The `update` function returns a new model with an `IsBusy = true` value (which can be to show an animation such as a spinner) as well as a command that actually calls the API
-* The MVU loop calls the command
-* The app continues to work normally - the spinner spins because `IsBusy = true`, and any other messages are processed as they would normally be. Note that you are of course free to process messages differently based on the fact that `IsBusy = true`; for example, you may choose to ignore additional `SignInRequested` messages.
-* When the API call returns, the function that called the API dispatches a suitable message based on the result (e.g. `SignInSuccessful` or `SignInFailed`)
+* The `update` function returns a new `model` with an `IsBusy = true` value (which can be shown as an animation such as a spinner) as well as a `Cmd<'msg>` function that calls the API asynchronously to perform the request and process the response onced invoked with a `dispatch` function.
+* MVU uses new model with view to update the UI and invokes the command by passing it a `dispatch` function.
```

Comment:
Rephrase: "The MVU framework updates the view using the new model and invokes the command by executing each function in the list with a `dispatch` function."